### PR TITLE
fix: pass className in Link

### PR DESCRIPTION
**Pull request by golota60**
Fixes: #1044

### CODE REVIEW

**Comment by golota60** [comment]
@jquense As a part of this, I've also inspected the output code between js build output of `1.1.2` and now, to ensure more silly stuff like this was not introduced. And it seems this + `children` not passed(which you fixed) were the only issues. I think we should do a release of this once merged

On the bright side, this reminds me that i think tests need a little bit of a refresher :smile:

File: src/foundReducer.ts [hunk 1]
```diff
@@ -21,12 +21,12 @@ const foundReducer = (
...
       // It doesn't make sense to resolve a match if there wasn't already an
       // unresolved match.
-      return state
-        ? {
-            match: state.match,
-            resolvedMatch: payload,
-          }
-        : null;
+      return (
+        state && {
```
Comment (jquense) [inline src/foundReducer.ts:25]: Can state be some non null value here ?

**Review by jquense — APPROVED** [review]

File: src/foundReducer.ts (hunk 1 above)
Comment (golota60) [reply src/foundReducer.ts:25]: `state` should never be null here afair, but since this is technically a functional change i reverted it too

**Comment by sushantdhiman** [comment]
Please release this fix @jquense

**Pull request merged**